src/arch/riscv/iommu.c

```diff
@@ -334,7 +334,10 @@ void rv_iommu_init(void)
     rv_iommu.hw.reg_ptr->fqh = 0;
 
     // Allocate IRQ for FQ
-    interrupts_reserve(platform.arch.iommu.fq_irq_id, rv_iommu_fq_irq_handler);
+    if(!interrupts_reserve(platform.arch.iommu.fq_irq_id, rv_iommu_fq_irq_handler)) {
+        ERROR("Failed to assign IOMMU FQ interrupt");
```

Comment:
```suggestion
        ERROR("Failed to reserve IOMMU FQ interrupt");
```

Reply:
Thank you, fixed in https://github.com/bao-project/bao-hypervisor/pull/97/commits/878ad2b2ed85b8ed383a63e32650ee9363c624d8.